istioctl/pkg/util/configdump/util.go

```diff
@@ -47,3 +47,17 @@ func (w *Wrapper) getSection(sectionTypeURL configTypeURL) (*anypb.Any, error) {
 
 	return dumpAny, nil
 }
+
+func (w *Wrapper) getSections(sectionTypeURL configTypeURL) ([]*anypb.Any, error) {
+	var dumpAny []*anypb.Any
```

Comment:
I think this is a little bit dup with `getSection`, can we reuse some codes?

Reply:
right now, `getSection` always return the last one.